src/main/java/bio/terra/workspace/service/workspace/flight/DeleteProjectStep.java

```diff
@@ -20,44 +23,48 @@
  * <p>Undo always fails for this step.
  */
 public class DeleteProjectStep implements Step {
```

Comment:
We need a way to express that this step should precede the delete context step. That knowledge is embedded in the undo logic but not really locked in.